app/templates/records/new_to_archives.html

```diff
@@ -16,8 +17,114 @@
 {% block content %}
 <div class="tna-section">
   <div class="tna-container">
-    <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
-      <h1 class="tna-heading-xl">{{ pageTitle }}</h1>
+    <div class="tna-column tna-column--width-3-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
+      <h1 class="tna-heading-l">Getting started at The National Archives</h1>
+      <p>If you are new to The National Archives and want some help getting started with your family history or any other kind of research, you're in the right place.</p>
+      <p>Use the sections below to find out where and how to starty your research or <a href="">explore curated highlights from our collection.</a>
+      </p>
+
+      <!--accordion-->
+      <div class="tna-accordion" data-multiselectable="false" data-module="tna-accordion">
+        <div class="tna-accordion__item" data-isopen="true">
+          <h3 class="tna-accordion__heading tna-heading-s">
```

Comment:
All the accordion headings should be `<h2>` level as there is no `<h2>` above them.

Reply:
Changed to `<h2>`